### ISSUE TITLE
peer review on CI lab9

### ISSUE THREAD

**Issue by martina-martini** [SimoneBorella/computational-intelligence#4]
I found the work about local search very interesting, particularly the choice of the hamming distance for the diversity evaluation. I appreciated the plots for all the problem instances. Still, I would like to suggest you a better representation of the graphs: in problem instance 1 the lines are very close to one another so they can result ambiguous. Moreover, the absence of the grid does not let the reader fully evaluate the results. The last advice I would like to leave to you regards the choice of the crossover mechanism: I would introduce a crossover rate, in order to introduce more randomness to your algorithm.
In conclusion, I found your work extremely complete. Well done!
